src/flamegraph/color/palettes.rs

```diff
@@ -437,6 +464,43 @@ mod tests {
         }
     }
 
+    #[test]
+    fn python_returns_correct() {
+        use super::python::resolve;
+
+        let test_names = [
+            TestData {
+                input: String::from("<frozen importlib._bootstrap>:_load_unlocked:680"),
+                output: BasicPalette::Green,
+            },
+            TestData {
+                input: String::from(".venv/lib/python3.9/time.py:12"),
+                output: BasicPalette::Green,
+            },
+            TestData {
+                input: String::from("C:/Users/User/AppData/Local/Programs/Python/Python39/lib/concurrent/futures/thread.py"),
```

Comment:
I'd like to see some Windows-style path tests here too :+1:

Reply:
Difficulty with that is that `Path` seems to only separate by backslash if you are actually on Windows - so the test fails on Linux. I'm not sure if there is a way to say "pretend you are on windows when running this test"

Reply:
We run the test suite on Windows too though, so then these tests should fail on Windows, no?

Actually, thinking some more about this, this would mean that if you captured a perf report on Linux and tried to debug it on Windows, it wouldn't work, which is a little concerning..

I wonder if we want to add a helper like:

```rust
fn split_any_path(path: &str) -> impl Iterator<Item = &str> {
    path.split(|c| c == '/' || c == '\')
}
```

Reply:
Ah I hadn't thought of that but yes - if you had a trace from Windows with backslashes in then you couldn't generate a graph on Linux.

I've added the helper you suggested - this also avoids having to deal with `OsString` so is better all round I think :)

Reply:
> We run the test suite on Windows too though, so then these tests should fail on Windows, no?

From the `Path` docs:
> This type supports a number of operations for inspecting a path, including breaking the path into its components (separated by / on Unix and by either / or \ on Windows)

So it makes sense that the current tests pass on windows - it's only going the other way which would be problematic